Jaeger: Add ability to upload JSON file for trace data (#37205)
grafana/grafana: docs/sources/datasources/jaeger.md

- **Data source -** Target data source.
- **Tags -** The tags that will be used in the Loki query. Default is `'cluster', 'hostname', 'namespace', 'pod'`.

> - **Span start time shift -** Shift in the start time for the Loki query based on the span start time. In order to extend to the past, you need to use a negative value. Time units can be used here, for example, 5s, 1m, 3h. The default is 0.

Reviewer: ```suggestion
- **Span start time shift -** Shift in the start time for the Loki query based on the span start time. In order to extend to the past, you need to use a negative value. Use time interval units like 5s, 1m, 3h. The default is 0.
```

- Max Duration - Filter all traces with a duration lower than the set value. Possible values are `1.2s, 100ms, 500us`.
- Limit - Limits the number of traces returned.

## Upload JSON trace file

> {{< figure src="/static/img/docs/explore/jaeger-upload-json.png" class="docs-image--no-shadow" caption="Screenshot of the Jaeger data source in explore with upload selected" >}}

Reviewer: Move the image to after the intro sentence in this section.

> You can upload JSON file that contains a single trace to visualize it. If the JSON contains multiple traces then the first one is going to be used. Here is an example JSON:

Reviewer: ```suggestion
You can upload a JSON file that contains a single trace to visualize it. If the file has multiple traces then the first trace is used for visualization. Here is an example JSON:
```

> You can upload JSON file that contains a single trace to visualize it. If the JSON contains multiple traces then the first one is going to be used. The JSON schema needs to look like this:

Reviewer 2: Could we link to the jaeger doc to be clear this is something you can download from jaeger-ui?

Reviewer 2: `The JSON schema needs to look like this`

Also as this is not a json schema but an example of the json I would probably reword it a bit.

Author: > Could we link to the jaeger doc to be clear this is something you can download from jaeger-ui?

I couldn't find anything that can be linked for this.
